CMakeLists.txt

```diff
@@ -264,9 +264,12 @@ endif()
 
 # spdlog
 add_library(spdlog INTERFACE)
-target_include_directories(spdlog INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/spdlog-20170710/include")
+target_include_directories(spdlog INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/spdlog-1.7.0/include")
 if (NOT WIN32)
 	set_property(TARGET spdlog APPEND PROPERTY INTERFACE_COMPILE_DEFINITIONS "SPDLOG_ENABLE_SYSLOG")
+else()
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DNOMINMAX")
```

Comment:
this could go to the top level, around line 111, as `add_definitions(-DNOMINMAX)`, I think

Reply:
They moved up there :)